src/test/scala/uk/gov/hmrc/test/ui/pages/registration/RegistrationPage.scala

```diff
@@ -376,13 +375,6 @@ object RegistrationPage extends BasePage {
     this
   }
 
-  def provideSecondContactDetailsYesOrNo(value: String): this.type = {
```

Comment:
how come we're removing this? Shouldn't there be at least one test to make sure that it works when we add a second contact?

Reply:
I have resused provideAddAnotherContactYesOrNo("No")

Reply:
and removed the 'provideSecondContactDetailsYesOrNo' method